tests/app/celery/test_scheduled_tasks.py

```diff
@@ -210,15 +214,17 @@ def test_should_not_update_status_of_letter_notifications(client, sample_letter_
 
 
 def test_timeout_notifications_sends_status_update_to_service(client, sample_template, mocker):
-    create_service_callback_api(service=sample_template.service)
+    callback_api = create_service_callback_api(service=sample_template.service)
```

Comment:
Should add a test for timeout_notifications where there is a technical_failure_notifications and verify that there is an exception thrown

Reply:
I think that's covered in test_provider_tasks.test_should_go_into_technical_error_if_exceeds_retries_on_deliver_sms_task and test_provider_tasks.test_should_go_into_technical_error_if_exceeds_retries_on_deliver_email_task